### ISSUE TITLE
Eslint traverses up from the project directory causing the extension to fail.

### ISSUE THREAD

**Issue by gatzjames** [Microsoft/vscode-eslint#175]
Reproduction Steps:
Lets say you open a project in vscode located at C:\users\username\project
with your eslint configuration file.
If for some reason you have an eslint configuration file in e.g. C:\users\username
the extension also tries to load that file's settings causing unexpected errors (missing plugins, config packages etc).

I don't know if this is a problem related to the extension or eslint, but i would be glad to help.

Also thanks for this great extension!

Additional Info:
OS: Windows 10,
vscode: 1.7.2,
Eslint settings: default extension settings

**Comment by dbaeumer** [comment]
This is the default ESLint library behavior and is not controlled by the extension. This also happens when you run eslint from the terminal in directory ` C:\users\username\project`. 

The extension shouldn't change this default ESLint library behavior.

**Comment by awestroke** [comment]
This should be reopened, as this is not default ESLint behaviour.

http://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy

Emphasis mine:
> Note: If you have a personal configuration file in your home directory (~/.eslintrc), **it will only be used if no other configuration files are found**. Since a personal configuration would apply to everything inside of a user’s directory, including third-party code, this could cause problems when running ESLint.

**Comment by awestroke** [comment]
I have the same problem running VSCode on Windows. The problem only happens in VS Code, not if I run ESLint from terminal.

**Comment by dbaeumer** [comment]
@awestroke can you share a GitHub repository demoing this. When I tried this eslint does travers up the parent directory chain until it finds a .eslintrc* file. If it doesn't find one it takes the one from the home directory.

**Comment by awestroke** [comment]
The problem is that even though vscode-eslint finds a .eslintrc file, it still uses the ~/.eslintrc file too. Both me and my colleagues have this problem, we all use Windows 10. The problem does not arise when using eslint from the command line.

**Comment by dbaeumer** [comment]
I was not able to reproduce this. I added 

```js
console.log('Hello World');

module.export = {
}
```
to my home directory and this is only executed if no other .eslintrc file is found in the parent chain. @awestroke could you please provide a GitHub respository I could clone that demos this.

**Comment by awestroke** [comment]
.eslintrc in project:
```json
{
}
```

.eslintrc in $HOME:
```json
{
    "extends": "foo"
}
```

Result from running eslint in terminal: no errors

Result from opening a js file in VS Code:
![2017-06-13 1116_23-index js eslint-vscode-dbg visual studio code](https://user-images.githubusercontent.com/1098408/27075300-00cfdccc-502a-11e7-9c54-596a32c31e5a.png)

**Comment by awestroke** [comment]
Can anyone else reproduce this?
